.github/workflows/tests.yml

```diff
@@ -48,22 +48,22 @@ jobs:
         with:
           cache-read-only: ${{ github.ref != 'refs/heads/develop' }}
 
-      - name: ⚙️ Run unit tests, debug and release
-        run: ./gradlew test $CI_GRADLE_ARG_PROPERTIES
+      - name: ⚙️ Run unit tests for debug variant
+        run: ./gradlew testDebugUnitTest $CI_GRADLE_ARG_PROPERTIES
```

Comment:
I am wondering if `./gradlew testGplayDebugUnitTest` would be enough.

Reply:
Probably. I kept it like that in case we add some `fdroid` only tests in the future. I'll change it and see if anything changes.

Reply:
Actually, that will only run `:app:testGplayDebugUnitTest` according to `--dry-run`, since the flavor only exists in that module.

Reply:
OK 👍